### ISSUE TITLE
Print all metrics after each epoch

### ISSUE THREAD

**Issue by bizzyvinci** [ThilinaRajapakse/simpletransformers#1149]
I'm working on `t5.t5_model('mt5', 'google/mt5-base')`. `running_loss` is displayed as the model is being trained, and after each epoch, the metrics (eval_loss, train_loss, custom_loss) is saved to *training_progress_scores.csv*. This is fine

However, I wish these metrics are displayed after each epoch instead of printing it after training has ended.

Or is that feature in Args already cause I can't find it.

[EDIT]
The solution to this issue is in [ModelArgs](https://simpletransformers.ai/docs/usage/#configuring-a-simple-transformers-model) as `evaluate_during_training_verbose.`

**Comment by vionwinnie** [comment]
Yeah I would like to pass the metrics to Wandb for monitoring. I wrote a function called calculate_f1(labels,pred) and then call 
model.train(train_df,eval_data=eval_df,f1=calculate_f1) however both calling the model.eval() or model.train() did not call upon the metrics calculation. I think we can work on improving the metrics component

**Comment by bizzyvinci** [comment]
Yes, looks like we going to do this ourself.

**Comment by bizzyvinci** [comment]
Hi @vionwinnie, which model in particular?

Cause the `train_model()` and `eval_model()` ought to call upon the metrics calculation.

Or maybe the solution is in [ModelArgs](https://simpletransformers.ai/docs/usage/#configuring-a-simple-transformers-model).
